examples/Tabs.main.tsx

```diff
@@ -3,33 +3,44 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { Tabs, Tab } from '@itwin/itwinui-react';
+import { Tabs } from '@itwin/itwinui-react';
 
 export default () => {
-  const [index, setIndex] = React.useState(0);
-  const getContent = () => {
-    switch (index) {
-      case 0:
-        return "Bentley Systems, Incorporated, is an American-based software development company that develops, manufactures, licenses, sells and supports computer software and services for the design, construction, and operation of infrastructure. The company's software serves the building, plant, civil, and geospatial markets in the areas of architecture, engineering, construction (AEC) and operations. Their software products are used to design, engineer, build, and operate large constructed assets such as roadways, railways, bridges, buildings, industrial plants, power plants, and utility networks. The company re-invests 20% of their revenues in research and development.";
-      case 1:
-        return 'Bentley Systems is headquartered in Exton, Pennsylvania, United States, but has development, sales and other departments in over 50 countries. The company had revenues of $700 million in 2018.';
-      default:
-        return 'Keith A. Bentley and Barry J. Bentley founded Bentley Systems in 1984. They introduced the commercial version of PseudoStation in 1985, which allowed users of Intergraphs VAX systems to use low-cost graphics terminals to view and modify the designs on their Intergraph IGDS (Interactive Graphics Design System) installations.';
-    }
-  };
   return (
-    <div style={{ paddingInline: 48 }}>
-      <Tabs
-        orientation='horizontal'
-        labels={[
-          <Tab key={1} label='Item1' />,
-          <Tab key={2} label='Item2' />,
-          <Tab key={3} label='Item3' />,
-        ]}
-        onTabSelected={setIndex}
-      >
-        {getContent()}
-      </Tabs>
-    </div>
+    <Tabs.Wrapper>
+      <Tabs.TabList>
+        <Tabs.Tab value='item1' label='Item1' />
+        <Tabs.Tab value='item2' label='Item2' />
+        <Tabs.Tab value='item3' label='Item3' />
+      </Tabs.TabList>
+
+      <Tabs.Panel value='item1'>
+        Bentley Systems, Incorporated, is an American-based software development
```

Comment:
nit: can we use more generic text? i think we were supposed to remove all company-specific content when we went open source, but somehow missed this one

better idea: use fruit names as tabs, with a description of the fruit as panel content

Reply:
Using generic text

Reply:
`value` can also be changed to `apple`, `orange`, etc

Reply:
Updated

Reply:
why `apple1`, `orange1`?

the uniqueness can be fixed by https://github.com/iTwin/iTwinUI/pull/1548#discussion_r1336392436